### PR TITLE
Fixed punctuation and removed unrelated sentence

### PR DESCRIPTION
**Pull request by sayedarifuddin**
(309) Removed unwanted comma and dot
(310) Changed single quotes to double quotes
(314) Removed unrelated sentence

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: wip/at-home/Whats-new-wip-at-home-20h1.md [hunk 1]
```diff
@@ -306,12 +306,11 @@ Please keep trying out this new IME, and let us know your feedback.
 Changing the picture for the account you use to sign in to Windows is now better than ever! Your updated picture will now show up quickly across Windows, apps, and many of the Microsoft sites you use every day. Previously, you might see as many as three or four different pictures for your account in different apps and sites at one time. Its super easy to try out – just go to Your info in Windows settings and change your picture there!
 
 __Steps__
-1. Open “Your info” in Windows settings by pressing the Windows key, and typing “Your info,” and selecting “Your account info.”
-2. Use the options in ‘Create your picture’ to select a new picture for your account.
+1. Open “Your info” in Windows settings by pressing the Windows key, and typing “Your info” and selecting “Your account info”
```
Comment (JohanFreelancer9) [inline wip/at-home/Whats-new-wip-at-home-20h1.md:309]:
```suggestion
1. Open “Your info” in Windows settings by pressing the Windows key, and typing “Your info” and selecting “Your account info.”
```

Comment (JohanFreelancer9) [reply wip/at-home/Whats-new-wip-at-home-20h1.md:309]: Originally, the period was used correctly.

Comment (sayedarifuddin) [reply wip/at-home/Whats-new-wip-at-home-20h1.md:309]: @JohanFreelancer9 Actually, if someone copy-paste ( Your info, ) to the Windows 10 Search, It doesn't show the intended result. The reason I removed the Comma. 
image

**Comment by sayedarifuddin** [comment]
> Copy review.

https://github.com/MicrosoftDocs/windows-insider/pull/293#discussion_r443146647

File: wip/at-home/Whats-new-wip-at-home-20h1.md (hunk 1 above)
Comment (JohanFreelancer9) [reply wip/at-home/Whats-new-wip-at-home-20h1.md:309]: Thanks for the explanation. Now it makes sense.

**Comment by mypil** [comment]
@bleblancMSFT - PR has been copy edited and is ready for final review.  

Thank you.

**Comment by yogkumgit** [comment]
@bleblancMSFT Please merge this PR. Thanks.

**Pull request merged**